### DISCUSSION TITLE
商榷整体大纲

### DISCUSSION THREAD

**Discussion by EvanLuo42** [FRC-China/FRC-Wiki#4] [Ideas]
在此商榷一下整体大纲。

**Comment by EvanLuo42** [comment]
当前板块分为：
- 关于本站
- FRC 简介
- 机械结构
- 电控系统
- 程序
- 最佳实践

如有安排不合理的地方可以提出。

**Reply by AlltAWD** [reply]
一个是希望“最佳实践”能够更具体一些？
以及是否应该加入一些非技术性内容，包括队伍管理，队伍宣传，赞助，赛前准备等等？

**Comment by leo-Zhizhu** [comment]
最佳实践部分的展现形式是什么？是希望做成独立的一章，去讲一些比较完整的案例还是说为前面几章抽象的知识提供一些例子然后link过去？
程序方面有具体的分类想法吗？是整体基于wpi还是说对于主流的ctre的代码，limelight以及python vision之类的都单独写一下？

**Reply by EvanLuo42** [reply]
最佳实践就是就是比如说如果我有一个吸物品的需求。因为这个可能在某个赛季出现过，且被迭代过很多次，完全可以直接拿来用。总而言之就是总结经验，方便其他人使用在自己的需求上。

**Reply by EvanLuo42** [reply]
程序我打算就是你说的这样，可能还会有一些算法之类的。

**Comment by leo-Zhizhu** [comment]
另外，目前是每一章都没有具体的细分目录吗？还是说有了但没放出来，对于这个目录的管理打算怎么做？分配到每个人或者每个队伍手上去处理某一章的具体内容划分会不会比较可行？前期有一个坚实的大纲才能方便后期的更新讨论

**Reply by AlltAWD** [reply]
感觉细分目录倒是可以等大纲，也就是一级目录确定下来再讨论。我认为分配到人来负责确实可行，但需要注意我还是希望能让所有人参与编辑，而不是像分配任务一样

**Reply by EvanLuo42** [reply]
> 另外，目前是每一章都没有具体的细分目录吗？还是说有了但没放出来，对于这个目录的管理打算怎么做？分配到每个人或者每个队伍手上去处理某一章的具体内容划分会不会比较可行？前期有一个坚实的大纲才能方便后期的更新讨论

每一章细分的目录得等大的板块出来之后才能定。

**Comment by AlltAWD** [comment]
突然想到，画cad应该放到哪，机械结构？听上去怪怪的
目前已知可以写教程的cad软件有：

- Inventor
- Onshape
- Solidworks
- Fusion360

**Reply by EvanLuo42** [reply]
可以单独开一个工具板块来写。

**Comment by richardyliu** [comment]
既然是和中国frc队伍有关的wiki，那是否可以加一个专门的非技术版块记录每支队伍的队伍简史？

**Reply by EvanLuo42** [reply]
这个似乎 Github 上有数据库项目，不应该囊括在 Wiki 里，内容太多了。

**Comment by EvanLuo42** [comment]
所以我们现在的结构应该是：
- 关于本站
- FRC 简介
- 机械（结构）
- 电控
- 程序（CTRE、算法、Photonvision）
- 工具（CAD 等软件使用）
- 金工（车床等的使用）
- 最佳实践（各种场景适用的解决方案归纳总结）

**Reply by AlltAWD** [reply]
那这样看来我还是觉得将“工具”与“金工”合并为一个大类，在下面进行细分比较好，比如“工具与软件”

**Reply by EvanLuo42** [reply]
那就这样：
- 关于本站
- FRC 简介
- 从零开始（如何建立一支队伍。以及如何管理）
- 机械（结构）
- 电控
- 程序（CTRE、算法、Photonvision）
- 工具与软件（CAD 等软件使用及金工内容）
- 最佳实践（各种场景适用的解决方案归纳总结）

**Reply by CLCK0622** [reply]
要不要git等程序相关的工具也放在这个板块下？

**Reply by EvanLuo42** [reply]
那就把git移到工具吧

**Comment by EvanLuo42** [comment]
看起来应该没有异议了，该讨论可以关闭了。
